editor/src/components/editor/store/editor-state.ts

```diff
@@ -565,14 +565,14 @@ export function modifyOpenParseSuccess(
   function getChanges(
     editor: EditorState,
     success: ParseSuccess,
-  ): ParseSuccessAndEditorChanges<{}> {
+  ): ParseSuccessAndEditorChanges<unknown> {
```

Comment:
hmmmmmm 🤔 

I think the `{}` had a semantic meaning to describe an _empty_ value. (I know in reality it was just a bad any) `unknown` feels off to me in this regards. I wonder if we could find something that is both correct and reads well

Reply:
Good shout. We could introduce a type to represent that:
```
type EmptyValue = {}
```
We could even do:
```
type EmptyValue = readonly {}
```
(I'm not sure if that actually adds anything)

Reply:
ouch

```typescript
const EmptyObjectO = ({} as const)
type EmptyObject = typeof EmptyObjectO
const cica: EmptyObject = 'hm' // this should fail but doesn't
```

it seems like the problem with {} runs deeper than I expected. An empty object literal is inferred to be `{}`. Which means "any non null value, including primitive values, not just object".

It seems like the best way to describe an empty object is this:

```typescript
const cica1: object = 'hm' // this fails as expected
const cica2: object = {} // this passes as expected
cica2.halo = 5 // Property 'halo' does not exist on type 'object'.
```

Reply:
Oooh looks like we have a winner in that we can do:
```
type EmptyObject = {[k: string]: never} 
const shouldPass: EmptyObject = {} // passes
const shouldFail1: EmptyObject = 'hm' // fails
const shouldFail2: EmptyObject = [] // fails
```

Reply:
nice!

Reply:
i'm going to go with `Record<string, never>`, sound good?